### PR TITLE
Move order_creator_finished signal etc. (SHOOP-2114)

### PR DESCRIPTION
**Pull request by suutari-ai**
* Move order_creator_finished signal from front to
  core.order_creator.signals
* Admin: Make all enabled shipping and payment methods available in
  order creation
* OrderCreator: Make request argument optional
  *  Remove request from OrderCreator init calls
  * Admin: Order creating: Populate Order.ip_address
* factories: Depend less on request
* Ensure pricing context gets shop and customer

First two of those commits are from @tulimaki (I only fixed the Refs format and rephrased change log item from "Enable all methods for order creator" to "Make all enabled shipping and payment methods available in order creator". The rest commits are related clean-ups/refactoring that we should do while moving the signal, since they fix a long standing TODO item and improves OrderCreator by making it request independent.

Replaces PR #351

### CODE REVIEW

File: shoop/core/order_creator/_creator.py [hunk 1]
```diff
@@ -23,8 +23,25 @@
...
+          for initializing remote address: If request is given and
+          `remote_address` is not, then `remote_address` will
+          initialized to ``request.META.get("REMOTE_ADDRESS")``.
+        :type remote_address: str
+        :param remote_address:
+          Remote IP address as string (IPv4 or IPv6).  If given, will be
+          stored to `~shoop.core.models.Order.ip_address` field of
+          created orders.
+        """
+        self.remote_address = (
```
Comment (akx) [inline shoop/core/order_creator/_creator.py]: Hmm. This should really be a property of the order source, not the creator.

Comment (suutari-ai) [inline shoop/core/order_creator/_creator.py]: Now that you bring that up... yeah.. I agree. Didn't occur to me... I'll move it.

Comment (suutari-ai) [inline shoop/core/order_creator/_creator.py]: It's now saved to OrderSource.ip_address and OrderCreator shouldn't be created with a request anymore.

**Comment by tulimaki** [comment]
@suutari-ai Thanks!

**Pull request merged**